sklearn/preprocessing/_encoders.py

```diff
@@ -699,6 +699,26 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    missing_value : object type or a list of object types, default "nan"
+        Which value(s) passed to ``transform()`` to be considered missing
+
+    handle_missing : string, default 'ignore'
+        How to handle missing values passed to ``transform()``
+
+        - 'ignore' : Allow missing values to pass through ``transform()`` and
```

Comment:
Maybe we should think of another name here? Because eg in https://github.com/scikit-learn/scikit-learn/issues/10465 I used the 'ignore' term in context of OneHotEncoder to be all zero (like unknown category), which probably is also not the best term for that use case, but still might indicate possible confusion.

Maybe something like 'passthrough' or 'preserve-nan' or ..

Reply:
That makes sense. I can do that.